test cases/vala/28 generated ui file/meson.build

```diff
@@ -0,0 +1,34 @@
+project('demo', 'c', 'vala')
+
+gnome = import('gnome')
+
+deps = [
+  dependency('glib-2.0', version : '>=2.50'),
+  dependency('gobject-2.0'),
+  dependency('gtk+-3.0'),
```

Comment:
Can we make this a GTK 4 test instead?

Reply:
Sure thing, I'll do it. I've actually adapted my toy example due to being unsure if GTK 4 is installed in CI (since all other Vala tests use GTK+3).

Reply:
Posting here for clarity: I've reverted to GTK3 for the time being and consider it to be a perfectly fine state of things. Messing with CI is wildly out-of-scope for this PR.